sepal_ui/scripts/gee.py

```diff
@@ -84,7 +84,9 @@ def is_running(task_descripsion: str) -> ee.batch.Task:
 
 
 @sd.need_ee
-def get_assets(folder: Union[str, Path] = "", asset_list: List[str] = []) -> List[str]:
+def get_assets(
+    folder: Union[str, Path] = "", asset_list: Optional[List[str]] = None
+) -> List[dict]:
```

Comment:
asset_list is a private parameter, it's use internally for recursion. I'll add an "_" and update the docstring accordingly

Reply:
so if that's the case I think is better if we add a nested function that does the recursion, don't you think?

Reply:
the nested function is more Pythonic because it doesn't expose the internal implementation details, and I would support that with the Zen of Python:

1. "explicit is better than implicit"
2. "There should be one-- and preferably only one --obvious way to do it."